ts/src/profiler.ts

```diff
@@ -159,13 +159,18 @@ function isBackoffResponseError(err: Error): err is BackoffResponseError {
  */
 export class Retryer {
   private nextBackoffMillis: number;
+
+  // For testing. Allows Math.random() to be replaced with non-random function.
+  private random: () => number;
+
   constructor(
       readonly initialBackoffMillis: number, readonly backoffCapMillis: number,
-      readonly backoffMultiplier: number) {
+      readonly backoffMultiplier: number, random = Math.random) {
```

Comment:
Is this Math.random change related to the source map fix? It is not obvious that it is.

Reply:
Yes -- adding a stub of Math.random() was the change which had previously caused tests with source maps to fail.